src/cleanlab_tlm/utils/chat_completions.py

```diff
@@ -5,8 +5,10 @@
 It works for any OpenAI LLM model, as well as the many other non-OpenAI LLMs that are also usable via Chat Completions API (Gemini, DeepSeek, Llama, etc).
```

Comment:
In VPC, what will be the recommended way to score:
- tool calls
- structured outputs

?